include/app_event_manager.h

```diff
@@ -40,21 +40,26 @@ extern "C" {
 typedef bool (*cb_fn)(const struct app_event_header *aeh);
 /**
  * @brief List of bits in event type flags.
+ *
+ *Flag description:
  */
 enum app_event_type_flags {
-	/* Flags set internally by Application Event Manager. */
+	/** APP_EVENT_TYPE_FLAGS_SYSTEM_START marks beginning of system flags.*/
 	APP_EVENT_TYPE_FLAGS_SYSTEM_START,
+	/** Flag set internally by Application Event Manager.
+	 *  APP_EVENT_TYPE_FLAGS_HAS_DYNDATA indicates that event type has dyndata data field.
+	 */
 	APP_EVENT_TYPE_FLAGS_HAS_DYNDATA = APP_EVENT_TYPE_FLAGS_SYSTEM_START,
-
-	/* Flags available for user. */
+	/** APP_EVENT_TYPE_FLAGS_USER_SETTABLE_START marks beginning of flags set by user.*/
 	APP_EVENT_TYPE_FLAGS_USER_SETTABLE_START,
+	/** Flag set by user.
+	 * APP_EVENT_TYPE_FLAGS_INIT_LOG_ENABLE enables event logging from the application start.
```

Comment:
In generated doc, this sentence is placed right below the enum value definition, so I think there is no need to duplicate the option name